trajopt/test/joint_costs_unit.cpp

```diff
@@ -73,7 +73,7 @@ TEST_F(CostsTest, equality_jointPos)  // NOLINT
   // Populate Basic Info
   pci.basic_info.n_steps = steps;
   pci.basic_info.manip = "right_arm";
-  pci.basic_info.start_fixed = false;
+  pci.basic_info.fixed_timesteps = { 0 };
```

Comment:
This line should be removed. Since it was original false the new fixed_timesteps are the indexes that should be fixed, so this should not have a zero index to align with the original start_fixed being false.